### ISSUE TITLE
Icon request: meeting

### ISSUE THREAD

**Issue by linguisten** [FortAwesome/Font-Awesome#13424]
### What can it be used for?
for meetings, conferences, etc. 

### Example image (optional)
image of a group of people around a table, maybe one presenting sth

### Icon request checklist

- [ x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [ x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x ] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [ ] I have included at least one example image if it could be helpful (optional)
- [ x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Does https://fontawesome.com/icons/users?style=solid fit your use case?

**Comment by linguisten** [comment]
No. On my forum i already used that one for "users", but i'd need something for "conference" or "meeting"

**Comment by spmsh** [comment]
upvote. meeting room icon would be greatly appreciated:

- image of a group of people around a table, maybe one presenting sth (as op explained)
- or table with chairs around and with/without a chalkboard behind

Some examples found on the google images search:
![meeting-icons](https://user-images.githubusercontent.com/36556605/47279979-acad8180-d606-11e8-911f-52900cf9c311.png)

Thanks !

**Comment by spmsh** [comment]
don't know it's ok to bump or not, but this is still highly required and appreciated (even if it's in pro version). Thanks!

**Comment by leifcr** [comment]
Please consider icons for attending meeting or registering for a meeting, for both webinars and physical meetings. These can likely be the same/similar.

**Comment by jampack** [comment]
bump

**Comment by guido-visser** [comment]
Is there any progress on this? This could be very handy definitely in Covid times

**Comment by spmsh** [comment]
Hi Team, just a bump as I'm following this one for quite some time now. Any progress/decision? Include it in the Pro bundle is perfectly fine. Thanks.

**Comment by spmsh** [comment]
bump allowed?

**Comment by spmsh** [comment]
the usual bump. this is much needed in the post pandemic word where employees only go to office for meetings.

**Comment by AlexPerez418** [comment]
Any progress here? 🤞🏽

**Comment by KaoriShingledecker** [comment]
I also have a suggested design, just a meeting table and chairs:
![image](https://github.com/FortAwesome/Font-Awesome/assets/149179852/3077fb1a-76fe-4835-9478-2bb511f6bd03)

**Comment by spmsh** [comment]
the yearly bump

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by spmsh** [comment]
the yearly bump (sorry I missed 2024)
